examples/sentinel/README.md

```diff
@@ -0,0 +1,97 @@
+# Sentinel Example
+
+This example shows how to write cost policies with Hashicorp's [Sentinel](https://www.hashicorp.com/sentinel).  For simplicity, this example evaluates policies with the Sentinel CLI (a.k.a. Sentinel Simulator).
```

Comment:
"For simplicity" makes me wonder what I should do in my actual workflow, should I not use this? or is there something else I should do? In general, docs that are explicit or more commanding are easier to follow.

Reply:
So from what I get, Sentinel is really tied to terraform enterprise, and the other hashi products (nomad).  It doesn't seem like it's meant to be a general purpose policy engine like OPA (though they clearly feel the need to compete with OPA).

Since they're really trying to push people away from the sentinel CLI, I think the point of this example is to show how a policy could be written against our json, not how to run sentinel.